src/main/java/bitflow4j/steps/database/ElasticsearchUtil.java

```diff
@@ -105,7 +105,12 @@ public void write(List<Sample> batch) throws IOException {
             bulkRequest.add(request);
         }
 
-        BulkResponse bulkResponse = client.bulk(bulkRequest, RequestOptions.DEFAULT);
+        BulkResponse bulkResponse = null;
+        try {
+            bulkResponse = client.bulk(bulkRequest, RequestOptions.DEFAULT);
+        } catch (IOException ioex){
+            logger.log(Level.SEVERE, "Internal error in elasticsearch.", ioex);
```

Comment:
If an exception is caught here, the bulkResponse variable will be null, so maybe handleResponse should not be called? Or can it handle null values?

Reply:
Ok I see that handleResponse does nothing when the input is null, could you still move the handleResponse() invokation into the try{} block? so its more clear that nothing happens when an exception is caught.